packages/resin-universal-gosquared/src/node.js

```diff
@@ -0,0 +1,38 @@
+var Promise = require('bluebird')
+var GoSquared = require('gosquared')
+
+module.exports = function(gosquaredId, apiKey, debug) {
+	var visitor = null
+	function createVisitor(userId) {
+		if (visitor) return visitor
+		var gosquared = new GoSquared({
+			api_key: apiKey,
+			site_token: gosquaredId
+		})
+
+		if (userId) {
+			gosquared.createPerson(userId)
```

Comment:
Ah I see what you mean... and after looking at the source you are ofc right, it's a factory and does not update the instance.